src/biogeophys/HydrologyNoDrainageMod.F90

```diff
@@ -278,7 +278,7 @@ subroutine HydrologyNoDrainage(bounds, &
            filter_hydrologyc, soilstate_inst, canopystate_inst, waterfluxbulk_inst, energyflux_inst)
       
       ! save the h2osoi_liq in top layer before evaluating the soilwater movement
-      if ( use_fan ) call store_tsl_moisture(waterstatebulk_inst) 
+      if ( use_fan ) call store_tsl_moisture(waterstatebulk_inst, filter_hydrologyc, num_hydrologyc) 
```

Comment:
Here and elsewhere: I find conditional blocks more readable if a true block is used - e.g.,

```Fortran
if (use_fan) then
   call store_tsl_moisture(...)
end if